packages/Welcome/exported/WelcomeDialog/language-generation/en-us/WelcomeDialog.en-us.lg

```diff
@@ -1,10 +1,10 @@
-# SendActivity_wStAtd()
+# SendActivity_PsAJ3p()
```

Comment:
We seem to be using the old form for SendActivity here:

![image](https://user-images.githubusercontent.com/12264946/111493199-b83bfe80-8713-11eb-8053-fa6c282b2298.png)

As opposed to response variations:

![image](https://user-images.githubusercontent.com/12264946/111493465-f46f5f00-8713-11eb-9718-8e134897a388.png)

which will generate an extra function:

![image](https://user-images.githubusercontent.com/12264946/111493657-27b1ee00-8714-11eb-95b4-dcdd1ee6afa9.png)


I think we should adopt the new response editor form in our packages and templates